docker-compose.yml

```diff
@@ -11,13 +11,16 @@ x-defaults:
     API_URL_SERVER: http://gateway:4000/graphql
     # Stripe
     STRIPE_SUBSCRIPTION_PRICE_ID: price_1IkVzxD5MZ3GejSORRBZCvK6
+    # Mailchimp
+    MAILCHIMP_LIST_ID: "12345abcdf89"
     # URLs
     ASSOCIATION_FRONTEND_URL: http://localhost:3020
     # Secrets
     PASTAPORTO_SECRET: pastaporto-secret
     IDENTITY_SECRET: identity-secret
     SERVICE_TO_SERVICE_SECRET: service-to-service-secret
     PASTAPORTO_ACTION_SECRET: pastaporto-action-secret
+    MAILCHIMP_SECRET_KEY: mailchimp-secret
```

Comment:
```suggestion
```

Reply:
for local development we could do something similar to what we did for `STRIPE_SECRET_API_KEY` for example, so if you want to enable the integration, you have to edit your `.env` file

Reply:
yeah but there are defaults for the other secrets, why remove this?

Reply:
the other secrets (SERVICE_TO_SERVICE_SECRET etc) have a meaning, for example the one I sent here it is the shared secret that services use to communicate.

`MAILCHIMP_SECRET_KEY: mailchimp-secret` means nothing, it will only make the pycon backend subscription code fail contacting mailchimp saying "the api key is not valid", 

that's the reason I told you to add this check

```python
    if not settings.MAILCHIMP_SECRET_KEY:
        raise ValueError("Mailchimp integration is not configured")
```

it is because it is a better DX error and makes clear to who sees it that they need to configure mailchimp if they want to test it.

If you add random values they would get random errors from mailchimp

Reply:
so what I am suggesting here is to do the same as how we configurated stripe (see there are no defaults for stripe?) so that it gets the value from .env

This allows a person to go to mailchimp and prepare the secret key, list id and so on so they can test the mailchimp integration and add their own values in .env

Reply:
okay

Reply:
but there is the STRIPE_SUBSCRIPTION_PRICE_ID , so should I left the LIst Id for Mailchimp??